tasks/main.yml

```diff
@@ -40,6 +40,16 @@
   apt_repository:
     repo: "deb https://repos.influxdata.com/{{ ansible_distribution.lower() }}/ {{ ansible_distribution_release  }} stable"
     state: present
+  when: (ansible_distribution|string == 'Ubuntu' and (ansible_distribution_release|string == 'focal' or ansible_distribution_release|string == 'bionic')) or 
+        ansible_distribution|string == 'Debian'
```

Comment:
We can remove this condition once stable release for `Ubuntu 22.04(jammy jellyfish)` is release on https://repos.influxdata.com/ubuntu/dists/